docs/reference/composites.rst

```diff
@@ -15,11 +15,12 @@ for using the D-Wave system.
 CutOffs
 =======
 
+Downsize the binary quadratic model (BQM) submitted to the child sampler by retaining
+only interactions with values commensurate with the sampler’s precision.
```

Comment:
Is there somewhere that discusses the precision in the system docs we can link to?

Reply:
See 2nd bullet in my last comment on #209